predictor/src/main/java/ir/ac/sbu/data_mining/mapper/TweetMapper.java

```diff
@@ -12,7 +12,13 @@ protected void map(Object key, Text value, Context context) throws IOException,
         String[] valArr = value.toString().split(" ");
         String emotion = valArr[2];
         String political = valArr[3];
+        int retweetCount = Integer.parseInt(valArr[1]);
         int rate = context.getConfiguration().getInt(emotion, 0);
+        int threshold = context.getConfiguration().getInt("threshold", 0);
+        if (retweetCount>threshold && threshold>0){
```

Comment:
Remove the condition. You can do this in this way
int coefficient = retweetCount / threshold;
rate *= (coefficient + 1)